src/parser/formatting-evaluator-module.ts

```diff
@@ -110,69 +107,67 @@ export class FormattingEvaluatorModule implements Module {
   _getFormattingScore(comment: GithubCommentScore) {
     // Change the \r to \n to fix markup interpretation
     const html = this._md.render(comment.content.replaceAll("\r", "\n"));
-    logger.debug("Will analyze formatting for the current content", { comment: comment.content, html });
+    logger.debug("Will analyze formatting for the current content:", { comment: comment.content, html });
     const temp = new JSDOM(html);
     if (temp.window.document.body) {
-      const res = this.classifyTagsWithWordCount(temp.window.document.body, comment.type);
-      return { formatting: res };
+      const res = this._classifyTagsWithWordCount(temp.window.document.body, comment.type);
+      return { formatting: res.formatting, regex: res.regex };
     } else {
       throw new Error(`Could not create DOM for comment [${comment}]`);
     }
   }
 
-  _countSymbols(regexes: FormattingEvaluatorConfiguration["multipliers"][0]["rewards"]["regex"], text: string) {
-    const counts: { [p: string]: { count: number; multiplier: number } } = {};
-    for (const [regex, multiplier] of Object.entries(regexes)) {
+  _countSymbolsFromRegex(
+    regexes: FormattingEvaluatorConfiguration["multipliers"][0]["rewards"]["regex"],
+    text: string
+  ) {
+    const counts: RegexCount = {};
+    for (const [regex, wordValue] of Object.entries(regexes)) {
       const match = text.trim().match(new RegExp(regex, "g"));
+      const wordCount = match?.length || 0;
+      const result = new Decimal(wordCount).pow(this._wordCountExponent).mul(wordValue).toDecimalPlaces(2).toNumber();
       counts[regex] = {
-        count: match?.length || 1,
-        multiplier,
+        wordCount,
+        wordValue,
+        result,
       };
     }
```

Comment:
regex doesn't necessarily mean words so this doesn't make sense to me

Reply:
We are currently only using it for word counting which I kind of hate. We should hardcode a word counter config property and then under the hood it just uses this regex. 

The repeated regex declarations in our configs looks bad and can be easily replaced with something like "word counter"

Reply:
Can be renamed to `regexCount` to make sense then.

Reply:
Can handle in a new pull but basically I'm suggesting to hard code this regex in place of a new config property so the config looks prettier.

Reply:
https://github.com/ubiquity-os-marketplace/conversation-rewards/pull/121/commits/19e70a4e946e8b6560a9265de95198c6c2d56b46